packages/ifpack/src/Ifpack_Hypre.cpp

```diff
@@ -415,38 +409,36 @@ int Ifpack_Hypre::Multiply(bool TransA, const Epetra_MultiVector& X, Epetra_Mult
 
 //==============================================================================
 std::ostream& Ifpack_Hypre::Print(std::ostream& os) const{
-  using std::endl;
-
   if (!Comm().MyPID()) {
-    os << endl;
-    os << "================================================================================" << endl;
-    os << "Ifpack_Hypre: " << Label () << endl << endl;
-    os << "Using " << Comm().NumProc() << " processors." << endl;
-    os << "Global number of rows            = " << A_->NumGlobalRows() << endl;
-    os << "Global number of nonzeros        = " << A_->NumGlobalNonzeros() << endl;
-    os << "Condition number estimate = " << Condest() << endl;
-    os << endl;
-    os << "Phase           # calls   Total Time (s)       Total MFlops     MFlops/s" << endl;
-    os << "-----           -------   --------------       ------------     --------" << endl;
+    os << std::endl;
```

Comment:
There's nothing wrong with a `using std::endl;` statement inside of a function body.  Why change this?  It just makes the diff bigger.

Reply:
The using statement didn't used to be there, that was a relatively recent addition (only 3 years ago!)  Most of this patch actually predates those changes by Alicia... (embarrassed face for not contributing it earlier).

Reply:
Done in next push.

Reply:
Alicia did a lot of truly thankless work for xSDK.  She's well beyond postdoc now and no longer at Sandia.